code/game/machinery/status_display.dm

```diff
@@ -175,6 +224,15 @@
 	dir = EAST
 	pixel_x = -32
 
+//makes it go on the wall when built
+/obj/machinery/status_display/Initialize(mapload, ndir, building)
+	. = ..()
+	if(building)
+		setDir(ndir)
+		pixel_x = (dir & 3)? 0 : (dir == 4 ? -32 : 32)
+		pixel_y = (dir & 3)? (dir ==1 ? -32 : 32) : 0
```

Comment:
Where is this copied and pasted from? These are all magic numbers and need to be replaced by the real directions, which I'm too tired to find out right now.

Reply:
this is from newscaster code